Prepare for v0.9.5

davidatwhiletrue opened the pull request:
### Summary
- Code documentation and documentation site configs.
- Integration and unit tests.
- Bugfix: deserialization of a deploy with StoredVersionedContractBy* executable deploy item failed because of a typo.
- Serialize BigIntegers as unsigned always.

### TODO

### Checklist

- [X] Code is properly formatted
- [X] All commits are signed
- [X] Tests included/updated or not needed
- [X] Documentation (manuals or wiki) has been updated or is not required

ihor commented on README.md:
````diff
@@ -15,25 +37,26 @@ dotnet build --configuration Release
 
 ```
-dotnet test --settings Casper.Network.SDK.Test/test.runsettings
+dotnet test --filter "TestCategory!~NCTL"
````
@davidatwhiletrue the command above fails becuse of double quotes on Windows:
```
$ dotnet test --filter "TestCategory!~NCTL"
zsh: event not found: ~NCTL
```

Can we changes it to single quotes? Do they work on Windows?

davidatwhiletrue replied:
Changed to single quotes.

ihor commented:
@davidatwhiletrue another thing I noticed is that we should add a link to the dockerized NCTL instructions in the README to make it easier to set up the test environment. Atm one should know how to use NCTL in order to proceed. Do you agree? Please make a note to add it once it's ready if you do.

davidatwhiletrue commented on README.md:
````diff
@@ -15,25 +37,26 @@ dotnet build --configuration Release
-* [Key management](./Docs/Articles/KeyManagement.md )
-* [Working with CLValue](./Docs/Articles/WorkingWithCLValue.md)
+```
+dotnet test --settings Casper.Network.SDK.Test/test.runsettings --filter "TestCategory~NCTL"
````
Changed to single quotes.

davidatwhiletrue commented:
@ihor It's a good idea. I've added to the README the command to start the container and a link to the full instructions. Similarly, I added a new page to the documentation website.
